lib/bundler/resolver/spec_group.rb

```diff
@@ -37,13 +39,17 @@ def activate_platform!(platform)
         @activated_platforms << platform
       end
 
+      def spec(platform)
```

Comment:
Another method name candidate is `for` because we use `for?(platform)`.
But `for` may confuse us because `for` is one of reserved word in Ruby.

Reply:
Maybe `spec_for`?

Note that we're already using a `for` method in `SpecSet`, so maybe `for` is ok here too. It probably won't be confusing if we make sure to always call it with a explicit receiver?